examples/chess/Cargo.toml

```diff
@@ -12,6 +12,7 @@ serde = "1.0"
 shakmaty = "0.22"
 
 [features]
-cuda = ["risc0-zkvm/cuda"]
 default = []
-metal = ["risc0-zkvm/metal"]
+cuda = ["risc0-zkvm/cuda"]
+prove = ["risc0-zkvm/cuda"]
```

Comment:
```suggestion
prove = ["risc0-zkvm/prove"]
```

Reply:
This issue appears to be duplicated in many examples

Reply:
Good catch. Fixed for all examples in 8b6f47e57f2fd9497532b4b1661075afd5a498b7